scripts/run-http-api-with-fixtures

```diff
@@ -52,43 +52,45 @@ prepare_workspace()
   echo "Unpacking fixture $FIXTURE"
   mkdir -p $WORKSPACE
   tar -xf $FIXTURE -C $WORKSPACE
+  sudo chown -R 0:0 $WORKSPACE
```

Comment:
This doesn't work on macOS.

Reply:
Yes, this is for ci only, I should add a flag or something.